### ISSUE TITLE
Aria-hidden elements are not focusable nor contain focusable elements

### ISSUE THREAD

**Issue by AlexSkrypnyk** [civictheme/uikit#76]
Focusable content should have tabindex='-1' or be removed from the DOM.

![60dadbc5-708a-4e5b-9752-2977c2d6a021](https://github.com/civictheme/uikit/assets/378794/a74c0ba0-ff46-45b5-bdec-fa86cf1bc9a0)

**Comment by AlexSkrypnyk** [comment]
This needs to be re-tested and more details are required. The Expand button is currently focusable and has a `title` attribute (`Expand Item 1 menu` etc.)

**Comment by AlexSkrypnyk** [comment]
This has been tested by @jcloys - looks like this is now resolved.
